src/ConnectedMode.UnitTests/Migration/ConnectedModeMigrationTests.cs

```diff
@@ -41,31 +39,28 @@ public void MefCtor_CheckTypeIsNonShared()
         public void MefCtor_CheckIsExported_NoProjectCleaners()
         {
             MefTestHelpers.CheckTypeCanBeImported<ConnectedModeMigration, IConnectedModeMigration>(
+                MefTestHelpers.CreateExport<IFileProvider>(),
+                MefTestHelpers.CreateExport<IFileCleaner>(),
                 MefTestHelpers.CreateExport<ILogger>());
         }
 
         [TestMethod]
-        public void MefCtor_CheckIsExported_MultipleProjectCleaners()
```

Comment:
We no longer need the old, more complicated test for `ImportMany` since we just import single instances now.